adapters/oidc/wildfly-elytron/src/main/java/org/keycloak/adapters/elytron/ElytronHttpFacade.java

```diff
@@ -77,6 +83,30 @@
     private SecurityIdentity securityIdentity;
     private boolean restored;
     private final Map<String, String> headers = new HashMap<>();
+    protected MultivaluedHashMap<String, String> queryParameters;
+
+    static {
+        boolean bogus = false;
+        String prop = System.getProperty("org.keycloak.adapters.elytronweb.forceBogus");
+        if (prop != null) {
+            bogus = Boolean.parseBoolean(prop);
+            log.tracef("Forcing value for bogus version of elytron-web %b", bogus);
+        } else {
+            try {
+                Class clazz = ElytronHttpFacade.class.getClassLoader().loadClass("org.wildfly.elytron.web.undertow.server.ElytronHttpExchange");
```

Comment:
@rmartinc Just out of curiosity. What's the intention behind the using this instead of `ElytronHttpExchange.class.getPackage().getImplementationVersion();` ?

Reply:
Not depending in the elytron-web module. If for whatever reason the module or class disappear we set the bogus property to false. I declared the module optional too in the module.xml.